mantle/cli/cli.go

```diff
@@ -65,7 +68,12 @@ func Execute(main *cobra.Command) {
 	})
 
 	if err := main.Execute(); err != nil {
-		plog.Fatal(err)
+		if err == ErrExitWarning77 {
+			plog.Warning(err)
+			os.Exit(77)
+		} else {
+			plog.Fatal(err)
+		}
```

Comment:
I'm starting to think we actually want to hide this behind a CLI flag like `--on-warn-failure-exit-77`. 

So locally if someone is running tests and a warn:true test fails they still get a passing exit code. But, in the pipeline where we want to do something different we'll pass in `--on-warn-failure-exit-77` and then check the exit code of 77.

Reply:
yep, i guess this switch would be better than forcing `77` by default. will add it